contentcuration/contentcuration/frontend/shared/views/contentNodeFields/CompletionOptions/ActivityDuration.vue

```diff
@@ -125,43 +144,49 @@
           return this.convertToMinutes(this.value);
         },
         set(value) {
-          this.handleUpdatedInput(value);
+          this.handleInput(value);
         },
       },
       maxRange() {
-        if (this.selectedDuration === DurationDropdownMap.SHORT_ACTIVITY) {
-          return SHORT_MAX;
-        } else if (this.selectedDuration === 'longActivity') {
-          return LONG_MAX;
+        if (this.isShortActivity) {
+          return SHORT_ACTIVITY_RANGE[SHORT_ACTIVITY_RANGE.length - 1];
+        } else if (this.isLongActivity) {
+          return LONG_ACTIVITY_RANGE[LONG_ACTIVITY_RANGE.length - 1];
         } else {
           return EXACT_MAX;
         }
       },
       minRange() {
-        if (this.selectedDuration === 'shortActvity') {
-          return SHORT_MIN;
-        } else if (this.selectedDuration === DurationDropdownMap.LONG_ACTIVITY) {
-          return LONG_MIN;
+        if (this.isShortActivity) {
+          return SHORT_ACTIVITY_RANGE[0];
+        } else if (this.isLongActivity) {
+          return LONG_ACTIVITY_RANGE[0];
```

Comment:
This looks like a change in behavior as `SHORT_ACTIVITY_RANGE[0] !== SHORT_MIN`. This is intentional?

Reply:
Yes, it was intentional, I saw that `minRange` and `maxRange` were not used in contexts where `selectedDuration` was different from `SHORT_ACTIVITY` or `LONG_ACTIVITY`, so the values of `SHORT_MIN`, `LONG_MIN`, `SHORT_MAX` and `LONG_MAX` were not being used anywhere so far, and I needed the actual min and max values of the selects options, so I updated them

Reply:
> so the values of SHORT_MIN, LONG_MIN, SHORT_MAX and LONG_MAX were not being used anywhere so far

I don't quite understand what you mean. It seems `SHORT_MIN` was being used?

```js
if (this.selectedDuration === 'shortActvity') {
  return SHORT_MIN;
}
```

Previously, the minimum for the long activity was 31, which is 1 minute more than the maximum for the short activity. Unless I'm misunderstanding, now with these changes it seems there's a gap between the short activity maximum and the long activity minimum? i.e. between 30 and 40 minutes

So a long activity that was 35 minutes, would result in a negative percentage on the slider?

Reply:
Yes, it was used there
```javascript
if (this.selectedDuration === 'shortActvity') {
  return SHORT_MIN;
}
```
but in the context of the computed property `this.minRange`.

But the `minRange` computed property was only used in this TextField, in the last conditional excecution:

```Vue
      <VFlex
        v-if="audioVideoUpload && selectedDuration === 'exactTime'"
      >
        ...
      </VFlex>
      <VFlex
        v-else-if="selectedDuration === 'shortActivity' || selectedDuration === 'longActivity'"
      >
       ...
      </VFlex>
      <VFlex v-else>
        <VTextField
          v-model="minutes"
          type="number"
          box
          :min="minRange" <!-- only here is used minRange and maxRange computed properties -->
          :max="maxRange"
          :label="$tr('minutesRequired')"
          :rules="minutesRules"
        />
      </VFlex>
    </VLayout>
```

And, that TextField is not shown when `selectedDuration === 'shortActivity' || selectedDuration === 'longActivity'`, so these conditions are never true:

```javascript
if (this.selectedDuration === 'shortActvity') {
  return SHORT_MIN;
} else if (this.selectedDuration === DurationDropdownMap.LONG_ACTIVITY) {
  return LONG_MIN;
}
```

So thats why I say that the values of SHORT_MIN, LONG_MIN, SHORT_MAX and LONG_MAX were not being used anywhere

Reply:
The valid values for short activity and long activity durations are just set by `SHORT_ACTIVITY_RANGE` and `LONG_ACTIVITY_RANGE` arrays

Reply:
Okay, I think it would be great to align the validators with this then: `getShortActivityDurationValidators` and `getLongActivityDurationValidators`